assets/style/style.css

```diff
@@ -0,0 +1,969 @@
+  gap: 3rem;
+  padding: 3rem 0;
+}
+
+.programs-title {
+  color: white;
+}
+
+.programs-title,
+.top-movies-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
+}
+
+.programs-title hr,
+.top-movies-title hr {
+  width: 3rem;
+  border: 1px solid #ff5438;
+}
+
+.programs-list {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.program {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: rgba(255, 255, 255, 0.1);
+  border: 1px solid rgba(255, 255, 255, 0);
+  transition: 0.5s ease;
+  padding: 0.5rem;
+}
+
+.program:hover {
+  background: rgba(255, 255, 255, 0.047);
+  border: 1px solid white;
+  cursor: pointer;
+  transition: 0.5s ease;
+}
+
+.program-icon {
+  height: 50px;
+  width: 50px;
+}
+
+.program-icon img {
+  object-fit: contain;
+  height: 100%;
+}
+
+.program-tag,
+.movie-date {
+  color: #ff5438;
+}
+
+.program-about {
+  width: 60%;
+}
+
+.more-prog {
+  cursor: pointer;
+  align-self: center;
+  background-color: #ef513c;
+  color: #d3d3d3;
+  padding: 1rem 3rem;
+}
+
+.top-movie-1 {
+  display: flex;
+  flex-direction: column;
+  gap: 0.3rem;
+}
+
+.more-movies {
+  cursor: pointer;
+  text-align: center;
+  padding: 1rem;
+  border: 1px solid #d3d3d3;
+}
+
+.top-movies-list {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  overflow: hidden;
+  height: 35rem;
+  transition: 0.5s ease;
+}
+
+.top-movies-list.expand {
+  height: max-content;
+  transition: 0.5s ease;
+}
+
+.movi-cover-name {
+  display: flex;
+  gap: 0.5rem;
+}
+
+.movie-name {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  width: 45%;
+}
+
+.movie-cover {
+  position: relative;
+  width: 10rem;
+  height: 10rem;
+}
+
+.movie-bg,
+.movie-img {
+  position: absolute;
+}
+
+.movie-bg {
+  top: 0;
+  left: 0;
+  height: 5rem;
+  width: 5rem;
+
+  /* background-image: url(/assets/img/b&w.jpg);
+  background-repeat: no-repeat;
+  background-size: cover; */
+}
+
+.movie-bg img {
+  object-fit: cover;
+  height: 100%;
+}
+
+.movie-img {
+  bottom: 0;
+  right: 0;
+  height: 9rem;
+  width: 9rem;
+}
+
+.movie-img img {
+  object-fit: cover;
+  height: 100%;
+}
+
+.top-movies {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+
+  /* padding: 2rem 0.5rem; */
+  border-bottom: 1px solid #dfdfdf;
+}
+
+.partners-section {
+  background-color: #272a31;
+  color: #d3d3d3;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.partner-list {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.partner {
+  display: flex;
+
+  /* flex-direction: column; */
+  align-items: center;
+  justify-content: center;
+  height: 7rem;
+  gap: 7%;
+}
+
+.partner img {
+  object-fit: contain;
+  height: 70%;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin-top: 1rem;
+  padding: 2rem 1rem;
+  color: #939393;
+  font-family: Lato-thin, serif;
+}
```

Comment:
- [ ] As the previous reviewer suggested,  we need to ensure that we stick to the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as much as possible. This will enable us to make the website more professional and user-friendly. Therefore I suggest that you ensure the background color of the `about page` has a dark color. This is what the previous reviewer was trying to refer to.

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/226062517-881503b4-24de-45e9-b0e3-7c45148a7284.png)

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226062657-4784a216-45f6-4c77-b170-794af01e8cc5.png)

Reply:
Thanks for your feedback dear @AlphaNtihinduka 
_can notice both pages has different background color in the footer._